javagen/src/main/java/com/azure/autorest/template/ClientMethodTemplate.java

```diff
@@ -247,23 +247,24 @@ public final void write(ClientMethod clientMethod, JavaType typeBlock) {
 
 
         boolean isFluentDelete = settings.isFluent() && restAPIMethod.getName().equalsIgnoreCase("Delete") && clientMethod.getMethodRequiredParameters().size() == 2;
+        typeBlock.javadocComment(comment -> {
+            comment.description(clientMethod.getDescription());
+            for (ClientMethodParameter parameter : clientMethod.getParameters()) {
+                comment.param(parameter.getName(), parameter.getDescription());
+            }
+            if (clientMethod.getParametersDeclaration() != null && !clientMethod.getParametersDeclaration().isEmpty()) {
+                comment.methodThrows("IllegalArgumentException", "thrown if parameters fail the validation");
+            }
+            if (restAPIMethod.getUnexpectedResponseExceptionType() != null) {
+                comment.methodThrows(restAPIMethod.getUnexpectedResponseExceptionType().toString(), "thrown if the request is rejected by server");
+            }
+            comment.methodThrows("RuntimeException", "all other wrapped checked exceptions if the request fails to be sent");
+            comment.methodReturns(clientMethod.getReturnValue().getDescription());
+        });
```

Comment:
Is it possible to pull it out to a shared method? Fluentgen might need to share it.

Reply:
@weidongxu-microsoft Done in [this commit](https://github.com/Azure/autorest.java/pull/480/commits/1a427c7e2da522fa0dd6d2ad8998948abfc8dc9a)